apps/desktop/src/renderer/routes/_authenticated/_dashboard/v2-workspace/$workspaceId/hooks/usePaneRegistry/components/DiffPane/components/CommentThread/comment-thread.css

```
@@ -0,0 +1,141 @@
+.diff-comment-body {
+	font-size: 13px;
+	line-height: 1.55;
+	color: var(--foreground);
+	overflow-wrap: anywhere;
+	word-break: break-word;
```

Stylelint (17.10.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)